src/App/pages/RepoPage/components/RepoInfo/RepoInfo.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import React from 'react';
+import { EyeIcon, ForkIcon, StarIcon, Token } from 'components/';
+import { FullRepositoryDisplayProps } from '../../types';
+import HomePageLink from './components/HomePageLink';
+import IconStat from './components/IconStat';
+import LanguageStats from './components/LanguageStats';
+import UserList from './components/UserList';
+import style from './RepoInfo.module.scss';
+
+export type RepoInfoProps = FullRepositoryDisplayProps & {
+  className?: string;
+};
+
+const RepoInfo: React.FC<RepoInfoProps> = ({
+  className,
+  homepage,
+  forks,
+  stars,
+  watchers,
+  topics,
+  languages,
+  contributors,
+  contributorsCount,
+}) => {
+  return (
+    <div className={classNames(style.container, className)}>
+      {homepage && <HomePageLink href={homepage} />}
+
+      {topics && <div className={style.topics}>{topics?.map((el, i) => <Token key={i}>{el}</Token>)}</div>}
```

Comment:
тоже ? лишний
```suggestion
      {topics && <div className={style.topics}>{topics.map((el, i) => <Token key={i}>{el}</Token>)}</div>}
```
если вдруг массив окажется пустым, то ничего не сломается из-за map, просто ничего не произойдёт